modules/enable-monitoring.adoc

```diff
@@ -0,0 +1,79 @@
+// Module included in the following assemblies:
+//
+// * configuration/monitor-acs.adoc
+:_module-type: PROCEDURE
+[id="enable-monitoring_{context}"]
+= Enabling monitoring
+
+Before you can monitor {product-title} you must enable monitoring.
+
+.Procedure
+. Patch the services to expose the port number `9090`.
+.. Patch the Sensor service:
+* On {ocp}:
++
+[source,terminal]
+----
+$ oc -n stackrox patch svc/sensor -p '{"spec":{"ports":[{"name":"monitoring","port":9090,"protocol":"TCP","targetPort":9090}]}}'
+----
+* On Kubernetes:
++
+[source,terminal]
+----
+$ kubectl -n stackrox patch svc/sensor -p '{"spec":{"ports":[{"name":"monitoring","port":9090,"protocol":"TCP","targetPort":9090}]}}'
+----
+.. Patch the Central service:
++
+[source,terminal]
+----
+$ oc -n stackrox patch svc/central -p '{"spec":{"ports":[{"name":"monitoring","port":9090,"protocol":"TCP","targetPort":9090}]}}'
+----
+. Modify network policies to allow ingress.
+* On {ocp}:
++
+[source,terminal]
+----
+$ oc apply -f - <<EOF
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  labels:
+    app.kubernetes.io/name: stackrox
+  name: allow-monitoring
+  namespace: stackrox
+spec:
+  ingress:
+  - ports:
+    - port: 9090
+      protocol: TCP
+  podSelector:
+    matchExpressions:
+    - {key: app, operator: In, values: [central, sensor, collector]}
+  policyTypes:
+  - Ingress
+EOF
+----
+* On Kubernetes:
```

Comment:
Not critical, but is `oc` vs. `kubectl` the only difference between these OpenShift vs Kubernetes commands? Just wondering if it would be better/more concise to simply add a note saying that for Kubernetes, use `kubectl` in place of `oc`.  Not something you have to change, just a thought.

Reply:
For most of the commands this is the case it would run exactly the same for both K8s and OCP. However in some cases OCP has some extra steps. Since all this content would appear as tabs in our docs it was fine earlier.

I've been separating K8s and OCP commands for all the PRs I've already done. Maybe I can change it for this one and any new ones that I'll create. And later after the docs are released I can go ahead and convert other ones.